AdyenDropIn/Presentation/WrapperViewController.swift

```diff
@@ -63,14 +74,21 @@ extension ModalViewController {
 
     private var leastPresentableHeightScale: CGFloat { 0.25 }
     private var greatestPresentableHeightScale: CGFloat {
-        UIApplication.shared.statusBarOrientation.isPortrait ? 0.9 : 1
+        UIDevice.current.userInterfaceIdiom == .phone && UIDevice.current.orientation.isPortrait ? 1 : 0.9
     }
 
     /// Enables any `UIViewController` to recalculate it's conten's size form modal presentation ,
     /// e.g `viewController.adyen.finalPresentationFrame(in:keyboardRect:)`.
     /// :nodoc:
     func finalPresentationFrame(in containerView: UIView, keyboardRect: CGRect = .zero) -> CGRect {
         var frame = containerView.bounds
+
+        if UIDevice.current.userInterfaceIdiom == .pad {
+            let width = min(frame.width * 0.85, 375 * UIScreen.main.scale)
```

Comment:
(85 % of width or 375) is repeated in two places, can we put this in one place, like a global function or something?

Reply:
Done in `Dimentions` enum